app/controllers/ems_common.rb

```diff
@@ -9,11 +9,13 @@ module EmsCommon
     # This is a temporary hack ensuring that @ems will be set.
     # Once we use @record in place of @ems, this can be removed
     # together with init_show_ems
-    alias_method :init_show_generic, :init_show
-    alias_method :init_show, :init_show_ems
+    alias_method :init_show_generic, :init_show if defined?(init_show)
+    alias_method :init_show, :init_show_ems if defined?(init_show_ems)
 
-    helper_method :textual_group_list
-    private :textual_group_list
+    if defined?(textual_group_list)
+      helper_method :textual_group_list
+      private :textual_group_list
+    end
```

Comment:
This is it. You are adding exception to a stuff that is included into far too many places :-(

My red light is on.